docs/site/src/demos/chips/Chips.js

```diff
@@ -0,0 +1,56 @@
+// @flow weak
+
+import React, { PropTypes } from 'react';
+import { createStyleSheet } from 'jss-theme-reactor';
+import Avatar from 'material-ui/Avatar';
+import Chip from 'material-ui/Chip';
+import Face from 'material-ui//svg-icons/action/face';
+
+const styleSheet = createStyleSheet('Chips', () => ({
+  chip: {
+    margin: 6,
```

Comment:
I think that it would be good to have a theme property for that. We could use it on the `Layout` component.